c2corg_ui/static/js/simplesearch.js

```diff
@@ -89,12 +93,21 @@ app.module.directive('appSimpleSearch', app.simpleSearchDirective);
  * @param {!angular.Scope} $scope Angular scope.
  * @param {angular.$compile} $compile Angular compile service.
  * @param {angular.$templateCache} $templateCache service
+ * @param {app.Document} appDocument service
+ * @param {app.Authentication} appAuthentication service
  * @param {angular.Attributes} $attrs Angular attributes.
  * @param {string} apiUrl Base URL of the API.
  * @param {angularGettext.Catalog} gettextCatalog Gettext catalog.
  * @ngInject
  */
-app.SimpleSearchController = function($scope, $compile, $attrs, apiUrl, gettextCatalog, $templateCache) {
+app.SimpleSearchController = function(appDocument, $scope, $compile, $attrs, apiUrl,
+                                                                            gettextCatalog, $templateCache, appAuthentication) {
```

Comment:
OK, but anyway we don't need that much indent. I think that for all other line breaks in arguments lists we used a 4-space indent like here: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/user.js#L39-L40